app/views/order_mailer/received.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Order Summary</h1>
+<% @order.line_items.each do |line_item| %>
+  <%= image_tag attachments["#{line_item.id}_first.jpg"].url %>
+  <h2>Product: <%= line_item.deal.title %></h2>
```

Comment:
prevent n+1 queries.
Add a gem called bullet and see n+1 queries and prevent them
You can do this in a new PR.

Reply:
discussed